requirements_dev.txt

```diff
@@ -0,0 +1,9 @@
+pip==20.1.1
```

Comment:
Just seeing this file here now. I was about to add quality and other checks, but I will probably wait until this merges and rebase. 

In a future PR we should move to the dependency management mentioned here: https://open-edx-proposals.readthedocs.io/en/latest/oep-0018-bp-python-dependencies.html

It is more complicated but will allow others to get up to speed more quickly.

Also, if we need examples, we have the https://github.com/edx/edx-cookiecutters repo with example files.